cypress/integration/common/selectors-and-assertions.spec.ts

```diff
@@ -95,7 +95,7 @@ export const thenMyFocusMovesTo = (selector: string): void => {
 };
 
 export const givenIHaveSelected = (selector: string): void => {
-  cy.get(selector).check();
+  cy.get(selector).should("not.be.checked").check();
 };
```

Comment:
As for `whenIType`: make it explicit that the user is selecting a previously unselected field.  Will error if it is already checked.